### DISCUSSION TITLE
Add existing organization to my enterprise with coupon

### DISCUSSION THREAD

**Discussion by ivancaride95** [community/community#133548] [Enterprise]
### Select Topic Area

Question

### Body

Hello! i'm having some troubles adding an existing organization to my new enterprise which i recently created and where i have redeem a coupon code for free enterprise plan for 1 year. When I want to add my organization they tell the following: ""Enterprise needs 5 additional seats to add {organization}". Furthermore when i want to try a seat or member to my enterprise i can't, there is no option to do so....

**Comment by queenofcorgis** [comment]
Hey @ivancaride95 - thanks for posting in the community. We'll need to take a look at your specific account to see what is going on, would you mind [opening a support ticket](https://support.github.com/contact) please.

**Reply by ivancaride95** [reply]
Thanks! already did, my ticket number is: Ticket 2902190
